src/features/controlPanel/index.tsx

```diff
@@ -24,13 +24,17 @@ const ControlPanelButtonSx: Partial<SxProps<Theme>> = {
   fontWeight: 500,
 };
 
-export const ControlPanel = (): ReactElement => {
+interface ControlPanelProps {
+  sx?: Record<string, unknown>;
```

Comment:
The linter didn't catch this one, but this prop is entirely unused and should either be removed or converted to `SxProps<Theme>` type.